<ui version="4.0" >
 <class>DataFilterDialogTemplate</class>
 <widget class="QDialog" name="DataFilterDialogTemplate" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Data filter</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Field:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="field_" >
     <property name="maximumSize" >
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text" >
       <string>Intensity</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Quality</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Charge</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Size</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Meta data</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="meta_name_label_" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Meta name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="meta_name_field_" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="maximumSize" >
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Operation:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="op_" >
     <property name="maximumSize" >
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text" >
       <string>>=</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>=</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>&lt;=</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>exists</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="value_label_" >
     <property name="text" >
      <string>Value:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="value_" >
     <property name="maximumSize" >
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>247</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>96</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button_" >
       <property name="text" >
        <string>Ok</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button_" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>field_</tabstop>
  <tabstop>meta_name_field_</tabstop>
  <tabstop>op_</tabstop>
  <tabstop>value_</tabstop>
  <tabstop>ok_button_</tabstop>
  <tabstop>cancel_button_</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_button_</sender>
   <signal>clicked()</signal>
   <receiver>DataFilterDialogTemplate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>437</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel" >
     <x>298</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
